idl/sprot.idol

```diff
@@ -22,7 +22,7 @@ Interface(
             idempotent: true,
         ),
         "rot_state": (
-            doc: "Return boot info about the RoT",
+            doc: "Return boot time info about the RoT",
```

Comment:
maybe `boot-time` info, to avoid ambiguity?

(also, should this be marked as deprecated?)

Reply:
Will do

Reply:
done